### PR TITLE
Improvements to the README file

### PR DESCRIPTION
**Pull request by atosatto**
Various improvements to the role documentation.

### CODE REVIEW

**Comment by pieterlexis** [comment]
Afaik, the correct spelling is 'dnsdist` (all lowercase, "like the unix-command") :smile:

File: README.md [hunk 1]
```diff
@@ -1,148 +1,181 @@
...
+
+The role also supports custom repositories
+
+```yaml
+- hosts: all
+  vars:
+    dnsdist_install_repo:
+      name: "dnsdist" # the repository name
+      apt_repo_origin: "my.repo.com"  # used to pin dnsdist to the provided repository
+      apt_repo: "deb http://my.repo.com/{{ ansible_distribution | lower }} {{ ansible_distribution_release | lower }}/dnsdist main"
```
Comment (pieterlexis) [inline README.md]: `example.com` is explicitly meant for documentation

File: README.md [hunk 2]
```diff
@@ -1,148 +1,181 @@
...
+      gpg_key: "http://my.repo.com/MYREPOGPGPUBKEY.asc" # repository public GPG key
+      gpg_key_id: "MYREPOGPGPUBKEYID" # to avoid to reimport the key each time the role is executed
+      yum_repo_baseurl: "http://my.repo.com/centos/$basearch/$releasever/dnsdist"
+      yum_debug_symbols_repo_baseurl: "http://repo.powerdns.com/centos/$basearch/$releasever/dnsdist/debug"
```
Comment (pieterlexis) [inline README.md]: you might wanna set this to the custom domain as well.

File: README.md (hunk 1 above)
Comment (atosatto) [reply README.md]: Done! :)

File: README.md (hunk 2 above)
Comment (atosatto) [reply README.md]: Right!

**Pull request merged**